radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/PendingVotes.java

```diff
@@ -54,18 +54,20 @@
 	// Make sure equals tester can access.
 	static final class PreviousVote {
 		private final View view;
+		private final long epoch;
 		private final HashCode hash;
 		private final boolean isTimeout;
 
-		PreviousVote(View view, HashCode hash, boolean isTimeout) {
+		PreviousVote(View view, long epoch, HashCode hash, boolean isTimeout) {

```

Comment:
no sure why is this needed? votes for other epochs are filtered out anyway in bft preprocessor, aren't they?